tests/unit/test_blob.py

```diff
@@ -4152,6 +4152,14 @@ def test_custom_time_setter(self):
         blob.custom_time = TIMESTAMP
         self.assertEqual(blob.custom_time, TIMESTAMP)
 
+    def test_custom_time_setter_none_value(self):
+        BLOB_NAME = "blob-name"
+        bucket = _Bucket()
+        TIMESTAMP = None
+        blob = self._make_one(BLOB_NAME, bucket=bucket)
+        blob.custom_time = TIMESTAMP
+        self.assertIsNone(blob.custom_time)
```

Comment:
This test should pre-populate the properties dict, e.g.:

```python
        TIMESTAMP = datetime.datetime(2014, 11, 5, 20, 34, 37, tzinfo=UTC)
        TIME_CREATED = TIMESTAMP.strftime(_RFC3339_MICROS)
        properties = {"customTime": TIME_CREATED}
        blob = self._make_one(BLOB_NAME, bucket=bucket, properties=properties)
```

Reply:
Please address @tseaver's comment.

Reply:
addressed.

Reply:
addressed.